### ISSUE TITLE
Applying logos images not working as expected

### ISSUE THREAD

**Issue by Gerschel** [Gerschel/sd-web-ui-quickcss#22]
@Nacurutu 
I was just testing if applying a new favicon would work, I get an error in the python console.  
>TypeError: join() argument must be str, bytes, or os.PathLike object, not 'list'  

This means there might be a change in how gradio handles the dropdown, or because of spaces in the filenames.  
I'm aware that dropdowns were having a problem before, and I had to create a workaround for them, so there is a chance they made
some updates to address it, which might've broken my workaround. It should've handled spaces in names gracefully, so I doubt that's the issue, I used built-in functions in python that are specifically for that and are system agnostic.  

I'm going to work on solving this right now. This post is just to make others aware of the issue if they run into this problem.

**Comment by Gerschel** [comment]
Umm. The problem disappeared on me when I went to debug it.  
So, I don't know what happened. I can't reproduce it.  
I did have to change from file type of png to svg, but that has zero affect on the issue I was having.  
I actually did not get an error before I made that change.  

But I think we are facing some other issue.  

Since they made a change to how files are instanced into the browser. Logos and favicons don't change on a soft reboot anymore, they change on a hard reboot.
